components/landing/TheHeroMoment/index.vue

```diff
@@ -74,80 +76,73 @@ export default class TheHeroMoment extends Vue {
   }
 
   &__container {
-    @include contained();
-
-    position: relative;
     background-image: url("/images/landing-page/hero-illustration.png");
     background-position: right center;
     background-repeat: no-repeat;
-    background-size: 70% auto;
-    padding-top: $layout-05;
+    background-size: contain;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    padding-top: $spacing-10;
     pointer-events: none;
 
-    @include mq($from: medium, $until: large) {
-      background-size: contain;
-    }
-
     @include mq($until: medium) {
       background-image: none;
     }
   }
 
   &__version-info {
-    margin: $layout-01 0 $layout-05;
+    @include mq($until: medium) {
+      order: 1;
+    }
 
-    @include mq($from: medium, $until: large) {
-      margin: 0 0 $layout-03;
+    @include mq($from: medium) {
+      margin-bottom: $spacing-07;
```

Comment:
Same, please replace this `$spacing-07` with `$layout-03`

Reply:
As discussed offline, we'll be replacing all `$layout` variables with `$spacing` variables with https://github.com/Qiskit/qiskit.org/issues/1789.